test/UnitTests/PlatformServices.Shared.Unit.Tests/netstandard1.3/ns13TraceListenerManagerTests.cs

```diff
@@ -70,7 +70,7 @@ public void DisposeShouldCallDisposeOnCorrespondingTraceListener()
             traceListenerManager.Add(traceListener);
             traceListenerManager.Dispose(traceListener);
 
-            // Tring to write after closing textWriter should throw exception
+            // String to write after closing textWriter should throw exception
```

Comment:
I think this is supposed to be Trying
```suggestion
            // Trying to write after closing textWriter should throw exception
```